Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt

```diff
@@ -185,13 +185,13 @@ fun EpisodeListItem(
                 )
                 top.linkTo(parent.top, 16.dp)
 
-                width = Dimension.preferredWrapContent
+                width = androidx.compose.foundation.layout.Dimension.preferredWrapContent
```

Comment:
Use import

Reply:
Nope, using this means we can suppress only the function, not the entire file.